cpp/src/Ice/SSL/SchannelTransceiverI.cpp

```diff
@@ -849,10 +881,29 @@ Schannel::TransceiverI::TransceiverI(
       _peerCertificate(nullptr),
       _remoteCertificateValidationCallback(serverAuthenticationOptions.clientCertificateValidationCallback),
       _certificate(nullptr),
-      _rootStore(nullptr),
+      _rootStore(serverAuthenticationOptions.trustedRootCertificates),
       _credentials({0, 0}),
-      _ssl({0, 0})
+      _ssl({0, 0}),
+      _chainEngine(nullptr)
 {
+    if (!_remoteCertificateValidationCallback)
```

Comment:
Schannel doesn't automatically validate client connections, we provide a default here.